README.md

```diff
@@ -59,6 +59,19 @@ See the INSTALL file for details; in brief, it uses the `configure`,
 check out the INSTALL file for some guidance on flags, or the output
 of `./configure --help`.
 
+An example configuration line that picks up packages installed at
+various paths might look like the following.  Note that depending on
+the package, the --with-<pkg>= directive expects slightly different
+things.  hdf5 wants a compiler wrapper; blas and lapack want lists of
+linker flags; elemental and gtest want an installation prefix.
+
+./configure --with-CXX=<mpipath>/bin/mpic++ \
+            --with-hdf5=<hdf5path>/bin/h5cc \
+	    --with-blas='-L<blaspath> -lopenblas' \
+	    --with-lapack='-L<blaspatch> -lopenblas' \
```

Comment:
"blaspatch" -> "lapackpath"

LAPACK doesn't necessarily have to share the BLAS path; Netlib's BLAS could be installed in a separate path from Netlib's LAPACK, for instance, or [BLIS](https://github.com/flame/blis) could be used with Netlib's LAPACK or the LAPACK provided by [libflame](https://github.com/flame/libflame). But some common LAPACK implementations also distribute a BLAS implementation (Netlib's LAPACK, OpenBLAS, MKL).